### PR TITLE
DEV-1516 updated the supertag icon. Added the icon prefix to svg files

### PR DESCRIPTION
**Pull request by Shwetank87**

### CODE REVIEW

File: dist/_kx-icon-font.scss [hunk 1]
```diff
@@ -17,25 +17,25 @@
...
 // Icons
-.kx-bar-chart:before { content: "\f101"; }
-.kx-cog:before { content: "\f102"; }
-.kx-cogs:before { content: "\f103"; }
-.kx-cross-device-w:before { content: "\f104"; }
-.kx-cross-device:before { content: "\f105"; }
-.kx-devices:before { content: "\f106"; }
-.kx-icon-cross-device:before { content: "\f107"; }
+.kx-cross-device-w:before { content: "\f101"; }
+.kx-cross-device:before { content: "\f102"; }
```
Comment (ethanph5) [inline dist/_kx-icon-font.scss]: Make sure they all start with `kx-icon-`

File: dist/_kx-icon-font.scss [hunk 2]
```diff
@@ -17,25 +17,25 @@
...
+.kx-icon-link:before { content: "\f10d"; }
+.kx-icon-notifications:before { content: "\f10e"; }
+.kx-icon-pin:before { content: "\f10f"; }
+.kx-icon-population:before { content: "\f110"; }
+.kx-icon-report-h:before { content: "\f111"; }
+.kx-icon-report:before { content: "\f112"; }
+.kx-icon-standard-segment-h:before { content: "\f113"; }
+.kx-icon-standard-segment:before { content: "\f114"; }
+.kx-icon-supertag:before { content: "\f115"; }
+.kx-mail:before { content: "\f116"; }
```
Comment (ethanph5) [inline dist/_kx-icon-font.scss]: Same here

**Comment by ethanph5** [comment]
LGTM, apart from the comments.

**Comment by NikitaSedyx** [comment]
@Shwetank87 I don't think you need change names for old svg files
or if you want to do it you should change all places in rb_ui

File: dist/_kx-icon-font.scss (hunk 1 above)
Comment (Shwetank87) [inline dist/_kx-icon-font.scss]: this is an old one. I did not introduce it. So it will break backwards compatibility

**Comment by NikitaSedyx** [comment]
:+1:

**Comment by Shwetank87** [comment]
thanks @NikitaSedyx  @ethanph5 

I'm going ahead and merging this. I've only changed names of newly introduced icons otherwise it will break backwards compatibility.

**Pull request merged**